setup/compiler.class.inc.php

```diff
@@ -2922,6 +2957,10 @@ protected function CompileThemes($oBrandingNode, $sTempTargetDir, $sFinalTargetD
 	 * @return string : file path of latest precompiled file to use for setup
 	 */
 	public function UseLatestPrecompiledFile(string $sTempTargetDir, string $sPrecompiledFileUri, $sPostCompilationLatestPrecompiledFile, $sThemeId) : ?string {
+		if (! utils::GetConfig()->Get('theme_precompilation_enabled')) {
```

Comment:
```suggestion
		if (! utils::GetConfig()->Get('theme.enable_precompilation')) {
```

Just renaming the param to be up to date with the proposition from previous commit